terraform/nixpkgs-tarballs.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_s3_bucket" "nixpkgs-tarballs" {
+  bucket = "nixpkgs-tarballs"
+  region = "eu-west-1"
+  acl    = "public-read"
+
+  website {
+    index_document = "index.html"
+  }
+}
```

Comment:
Note that this bucket is already managed by NixOps.

Reply:
What happens if it gets removed from NixOps, will NixOps try to destroy it?

Reply:
anyways, I just commented out the declaration for now